src/kfs_frontend/kfs_utils.cpp

```diff
@@ -176,4 +177,22 @@ void setBatchSize(KFSTensorOutputProto& proto, int64_t batch) {
 void setStringPrecision(KFSTensorOutputProto& proto) {
     proto.set_datatype("BYTES");
 }
+Status getRawInputContentsBatchSizeAndWidth(const std::string& buffer, int32_t& batchSize, size_t& width) {
+    size_t offset = 0;
+    size_t tmpBatchSize = 0;
+    size_t tmpMaxStringLength = 0;
+    while (offset + sizeof(uint32_t) <= buffer.size()) {
+        size_t inputSize = *((uint32_t*)(buffer.data() + offset));
```

Comment:
Are we sure at this point that data is of uint32_t?
Please use c++ style cast (reinterpret_cast)